src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1491,6 +1468,37 @@ void AutomationInstrumentClipView::editPadAction(bool state, uint8_t yDisplay, u
 	}
 }
 
```

Comment:
new function created as a result of refactoring above. this records a pad press into the editPadPress array struct.